subsys/bluetooth/controller/Kconfig

```diff
@@ -484,5 +484,10 @@ config BT_CTLR_SDC_SILENCE_UNEXPECTED_MSG_TYPE
 	help
 	  Convenience config to silence unexpected incoming msg type
 
+config BT_CTLR_SDC_IGNORE_HCI_ISO_DATA_TS_FROM_HOST
+	bool
```

Comment:
Should probably have a prompt: 
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/build/kconfig/tips.html#what-to-turn-into-kconfig-options